src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/ScheduleServiceTest.kt

```diff
@@ -334,6 +334,36 @@ class ScheduleServiceTest {
         .hasMessageContaining("Schedule for date $today has times out of order - 08:00 to 07:59")
     }
 
+    @Test
+    fun `should allow creation of a new schedule for today`() {
```

Comment:
Unrelated - missing date boundary test to prove that scheduled instances can be created for today.